packages/react/src/domain/handles.ts

```diff
@@ -52,6 +58,14 @@ export function isFactorOidc(factor: Factor): factor is FactorOIDC {
   return factor.method === "oidc";
 }
 
+export function isFactorEmailLink(factor: Factor): boolean {
+  return factor.method === "email_link";
+}
+
+export function isFactorSmsLink(factor: Factor): boolean {
```

Comment:
These would be a bit more useful in the form of type guards - can you check the existing `is*` functions in this module for inspiration?